draft-ietf-quic-recovery.md

```diff
@@ -481,19 +484,31 @@ DetectLostPackets takes one parameter, acked, which is the largest acked packet.
 Pseudocode for DetectLostPackets follows:
 
 ~~~
-   DetectLostPackets(acked):
+   DetectLostPackets(largest_acked):
```

Comment:
It's a packet, not a number, because I use fields of the packet below, such as the time it was sent.